main.go

```diff
@@ -43,8 +43,9 @@ Logging Options:
     -r, --remote_syslog <addr>       Syslog server addr (udp://localhost:514)
     -D, --debug                      Enable debugging output
     -V, --trace                      Trace the raw protocol
+    -VV                              Verbose trace (traces system account) 
     -DV                              Debug and trace
-    --sys_trace                      Trace the system account (default: false)
+    -DVV                             Debug and verbose trace (traces system account) 
```

Comment:
traves all accounts? The () comment might make it feel like only the system account.

Reply:
-VV                              Verbose trace (traces system account as well)

Reply:
LGTM